beacon-chain/core/epoch/epoch_operations_test.go

```diff
@@ -675,3 +669,51 @@ func TestInclusionDistance_NotFound(t *testing.T) {
 		t.Fatal("InclusionDistance should have failed")
 	}
 }
+
+func TestRandaoMix_Ok(t *testing.T) {
+	randaoMixes := make([][]byte, config.LatestRandaoMixesLength)
+	for i := 0; i < len(randaoMixes); i++ {
+		intInBytes := make([]byte, 32)
+		binary.BigEndian.PutUint64(intInBytes, uint64(i))
+		randaoMixes[i] = intInBytes
+	}
+	state := &pb.BeaconState{LatestRandaoMixesHash32S: randaoMixes}
+	tests := []struct {
+		slot      uint64
+		randaoMix []byte
+	}{
+		{
+			slot:      10,
+			randaoMix: randaoMixes[10],
+		},
+		{
+			slot:      2344,
+			randaoMix: randaoMixes[2344],
+		},
+		{
+			slot:      99999,
+			randaoMix: randaoMixes[99999%config.LatestRandaoMixesLength],
+		},
+	}
+	for _, test := range tests {
+		state.Slot = test.slot + 1
+		mix, err := randaoMix(state, test.slot)
+		if err != nil {
+			t.Fatalf("Could not get randao mix: %v", err)
+		}
+		if !bytes.Equal(test.randaoMix, mix) {
+			t.Errorf("Incorrect randao mix. Wanted: %#x, got: %#x",
+				test.randaoMix, mix)
+		}
+	}
+}
+
+func TestRandaoMix_OutOfBound(t *testing.T) {
```

Comment:
Naming? Should we use underscores or not use them? I dont feel strongly about this but my recent PR had feedback not to use underscores in the test functions

Reply:
I dont feel strongly either about underscores, but the large majority of out tests do not use them in their naming. It would be better to stick to one standard, otherwise our test naming will look pretty inconsistent across all the diff packages

Reply:
We've used them for all block operations - I think in tests it makes things clearer and easier to read when running go test -v

Reply:
Fair enough, let's stick to what we have across most of the repo

Reply:
I wouldn't say large majority. 110 out of 266 (41%) tests in under `beacon_chain` uses underscores, and I do agree with @rauljordan, it makes things clearer and easier to read when running go test -v

Reply:
looks like underscores are preferred in our test naming, maybe we can standardize this across the repo for all our tests ?